hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -2183,6 +2183,20 @@
     <description>Byte limit for Raft's Log Worker queue.
     </description>
   </property>
+  <property>
+    <name>ozone.om.ratis.pending.write.byte-limit</name>
+    <value>32MB</value>
+    <tag>OZONE, DEBUG, OM, RATIS</tag>
+    <description>Raft's Pending writes Byte limit.
```

Comment:
We can change the description to follow https://github.com/apache/ratis/blob/master/ratis-docs/src/site/markdown/configurations.md , same with the other configuration.

Reply:
done